Content.Client/Cargo/UI/CargoConsoleMenu.xaml.cs

```diff
@@ -210,10 +210,11 @@ public void PopulateOrders(IEnumerable<CargoOrderData> orders)
 
             foreach (var order in orders)
             {
-                if (order.Approved)
+                if (order.Approved ||
+                    !_protoManager.Resolve(order.Product, out var productProto))
                     continue;
 
-                var product = _protoManager.Index<EntityPrototype>(order.ProductId);
+                var product = _protoManager.Index<EntityPrototype>(productProto.Product);
                 var productName = product.Name;
```

Comment:
There's an inconsistency where the name of the order in the main list is different from the name in pending orders.

```suggestion
                var product = _protoManager.Index<EntityPrototype>(productProto.Product);
                var productName = productProto.Name;
```